recipe/install-static.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/lib
+cp \
```

Comment:
Not sure where these files live at this point of the build.

@jjhelmus what was your approach with `openssl`?

Reply:
Each output gets a copy of the work directory as it exists after the `build.sh` or `bld.bat` script runs.  The layout of this directory is going to depend on the tools and content of those files.  Adding a `exit 1` to the end of `build.sh` or using `conda debug` will give you chance to examine a copy of the folder.  

You can also examine the Makefile to see what `install` target does, most of the time it just copies various files into the prefix.  Sometimes there is a `make install-static` or similar target which makes the process easier.  

From my examination it looks like the static libraries (and the dynamic ones) are staged in `.libs` directories is various locations:

```
$ find . -name "libhdf*.a"
./c++/src/.libs/libhdf5_cpp.a
./fortran/src/.libs/libhdf5_fortran.a
./hl/c++/src/.libs/libhdf5_hl_cpp.a
./hl/fortran/src/.libs/libhdf5hl_fortran.a
./hl/src/.libs/libhdf5_hl.a
./src/.libs/libhdf5.a
```

Reply:
> Adding a exit 1 to the end of build.sh or using conda debug will give you chance to examine a copy of the folder.

Thanks!

>  Sometimes there is a make install-static or similar target which makes the process easier.

I'll check if that is the case here.

Reply:
> Sometimes there is a make install-static or similar target which makes the process easier.

I could not find anything, keeping the current approach.